openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk

```diff
@@ -14,17 +14,17 @@ export VERILOG_FILES 		= $(sort $(wildcard ./design/src/$(DESIGN_NICKNAME)/*.v))
 export CLK_PERIOD_IN_NS         = 1000
 export SDC_FILE    		= ./design/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
-export DIE_AREA   	 	= 0 0 280 380
-export CORE_AREA   		= 15 15 265 365
+export DIE_AREA   	 	= 0 0 340 340
```

Comment:
Have you been able to calculate this for each DLDO configuration? If so how?

Reply:
As of the the area is fixed. I tested the flow with various number of switches. The only parameter that should be changed is PLACE_DENSITY for varying number of switches. And that has been implemented in the flow.

Reply:
This should be automated based on the instances count out of synthesis and modeling. Can you please add it as a Github issue and assign it to yourself?

Reply:
Yes.